internal/services/workloads/workloads_sap_deployment_virtual_instance_resource.go

```diff
@@ -0,0 +1,2397 @@
+	LoadBalancer        []LoadBalancer                    `tfschema:"load_balancer"`
+	VirtualMachines     []VirtualMachineFullResourceNames `tfschema:"virtual_machine"`
+}
+
+type LoadBalancer struct {
+	BackendPoolNames             []string `tfschema:"backend_pool_names"`
+	FrontendIpConfigurationNames []string `tfschema:"frontend_ip_configuration_names"`
+	HealthProbeNames             []string `tfschema:"health_probe_names"`
+	Name                         string   `tfschema:"name"`
+}
+
+type SharedStorage struct {
+	AccountName         string `tfschema:"account_name"`
+	PrivateEndpointName string `tfschema:"private_endpoint_name"`
+}
+
+type WorkloadsSAPDeploymentVirtualInstanceResource struct{}
+
+var _ sdk.ResourceWithUpdate = WorkloadsSAPDeploymentVirtualInstanceResource{}
+
+func (r WorkloadsSAPDeploymentVirtualInstanceResource) ResourceType() string {
+	return "azurerm_workloads_sap_deployment_virtual_instance"
+}
+
+func (r WorkloadsSAPDeploymentVirtualInstanceResource) ModelObject() interface{} {
+	return &WorkloadsSAPDeploymentVirtualInstanceModel{}
+}
+
+func (r WorkloadsSAPDeploymentVirtualInstanceResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
+	return sapvirtualinstances.ValidateSapVirtualInstanceID
+}
+
+func (r WorkloadsSAPDeploymentVirtualInstanceResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.SAPVirtualInstanceName,
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"location": commonschema.Location(),
+
+		"deployment_with_os_configuration": {
+			Type:     pluginsdk.TypeList,
+			Required: true,
+			ForceNew: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"app_location": commonschema.Location(),
+
+					"os_sap_configuration": {
+						Type:     pluginsdk.TypeList,
+						Required: true,
+						ForceNew: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"sap_fqdn": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ForceNew:     true,
+									ValidateFunc: validate.SAPFQDN,
+								},
+
+								"deployer_virtual_machine_packages": {
+									Type:     pluginsdk.TypeList,
+									Optional: true,
+									ForceNew: true,
+									MaxItems: 1,
+									Elem: &pluginsdk.Resource{
+										Schema: map[string]*pluginsdk.Schema{
+											"storage_account_id": {
+												Type:         pluginsdk.TypeString,
+												Required:     true,
+												ForceNew:     true,
+												ValidateFunc: commonids.ValidateStorageAccountID,
+											},
+
+											"url": {
+												Type:         pluginsdk.TypeString,
+												Required:     true,
+												ForceNew:     true,
+												ValidateFunc: validation.IsURLWithHTTPorHTTPS,
```

Comment:
Is this `url` part of the storage account? Is it a blob or something else?

Reply:
It could be the url that is able to be accessed from anywhere.